sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -1284,25 +1281,23 @@ private synchronized void updateReleaseDetails(ReleaseDetails releaseDetails) {
 
     /**
      * Get reporting parameters for updated release.
-     *
-     * @param isPublic            are the parameters for public group or not.
-     *                            For public group we report install_id and release_id.
+     *  @param isPublic            are the parameters for public group or not.
+     *                            For public group we report install_id, distribution_group_id and release_id.
      *                            For private group we report distribution_group_id and release_id.
-     * @param distributionGroupId distribution group id.
      */
     @NonNull
-    private String getReportingParametersForUpdatedRelease(boolean isPublic, String distributionGroupId) {
+    private String getReportingParametersForUpdatedRelease(boolean isPublic) {
         String reportingParameters = "";
         AppCenterLog.debug(LOG_TAG, "Check if we need to report release installation..");
         String lastDownloadedReleaseHash = SharedPreferencesManager.getString(PREFERENCE_KEY_DOWNLOADED_RELEASE_HASH);
+        String lastDistributionGroupId = SharedPreferencesManager.getString(PREFERENCE_KEY_DOWNLOADED_DISTRIBUTION_GROUP_ID);
```

Comment:
This is actually being cleared and the main key is updated at start time. So we need to use the main distribution groupId key like before this PR.